src/sass/_common/reskin.scss

```diff
@@ -0,0 +1,1090 @@
+
+    button:focus {
+        background-color: var(--core-color-solid-coral-700);
+    }
+}
+
+.not-available-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: var(--core-spacing-1600);
+    max-width: 940px;
+}
+
+.not-available-section-text {
+    color: var(--component-textIcon-normal-prominent);
+}
+
+.static-page-loader {
+   margin-top: 15px;
+}
+
+.main-loader {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    max-width: 942px;
+    margin: 0 auto;
+
+
+    @media (max-width: 768px) {
+        padding: 16px;
+        
+    }
+    
+    .header-loader {
+        display: flex;
+        margin-top: 42px;
+        gap: 26px;
+
+        @media (max-width: 768px) {
+           flex-direction: column-reverse;
+           gap: 16px;
+            
+        }
+
+        .dropdown-loader-container {
+            display: flex;
+            gap: 12px;
+
+            @media (max-width: 768px) {
+                flex-direction: column;
+                   
+            }
+
+            .dropdown-loader {
+                width: 220px;
+
+                @media (max-width: 768px) {
+                    width: 100%;
+                       
+                }
+            }
+        }
+        .other-header-loader-container {
+            width: 100%;
+            display: flex;
+            justify-content: space-between;
+        }
+    }
+    .body-loader {
+        display: flex;
+        margin-top: 18px;
+        width: 100%;
+        gap: 16px;
+
+        @media (max-width: 768px) {
+            flex-direction: column;
+               
+        }
+
+        .trade-section-loader {
+            display: flex;
+            width: 100%;
+            max-width: 462px;
+
+            @media (max-width: 768px) {
+                max-width: 100%;
+                   
+            }
+        }
+        .purchase-section-loader {
+            display: flex;
+            flex-direction: column;
+            gap: var(--core-spacing-800);
+            width: 100%;
+            max-width: 490px;
+
+            @media (max-width: 768px) {
+                max-width: 100%;
+                   
+            }
+        }
+    }
+    .footer-loader {
+        display: flex;
+        width: 100%;
+        justify-content: center;
+        margin-top: 132px;
+    }
+}
+
+.static-page-layout {
+    margin-top: 15px;
+    margin-bottom: 20px;
+
+    @media (max-width: 768px) {
+        margin-top: 25px;
+        
+    }
+}
+
+.static-content {
+    margin : 0 auto;
+    max-width: 960px;
+    padding-inline: 10px;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+.input-class{
+    width: 346px !important;
+    padding: 10px 12px;
+    border: 1px solid var(--core-color-solid-slate-400);
+    font-size: 14px;
+    height: 40px; 
+
+    @media (max-width: 768px) {
+        height: 48px !important;
+        width: 312px !important;
+        
```

Comment:
1. Seems like an extra line break here:)
2. Can we apply a quill token instead of 48px
3. What is the reason to use `!important`?
5. Can we also add a line break at the end of the file please to avoid prettifier complaining about it🙏🏻?